### PR TITLE
notification title outside scrolling

### PR DESCRIPTION
**Pull request by SamueleA**
notification title is not part of scrolling

Peek 2020-11-19 10-22

### CODE REVIEW

File: .storybook/webpack.config.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ module.exports = ({ config }) => {
     '@events': path.resolve(process.cwd(), 'src', 'events'),
     '@utils': path.resolve(process.cwd(), 'src', 'utils'),
     '@reducers': path.resolve(process.cwd(), 'src', 'reducers'),
+    '@electron-store': path.resolve(process.cwd(), 'src', 'electron-store'),
```
Comment (giankotarola) [inline .storybook/webpack.config.js:12]: is there a `electron-store` file?

File: src/shared/components/Modal/Prompt/prompt.stories.js [hunk 2]
```diff
@@ -1,37 +1,38 @@
-import React from 'react';
-import { storiesOf } from '@storybook/react';
-import { text, object, boolean } from '@storybook/addon-knobs';
-import { action } from '@storybook/addon-actions';
+// TODO: fix <Prompt/> component for storybook
```
Comment (giankotarola) [inline src/shared/components/Modal/Prompt/prompt.stories.js:1]: perhaps just remove this.

File: .storybook/webpack.config.js (hunk 1 above)
Comment (giankotarola) [reply .storybook/webpack.config.js:12]: oh i see the the folder now

**Review by giankotarola — APPROVED** [review]

File: src/shared/components/Modal/Prompt/prompt.stories.js (hunk 2 above)
Comment (SamueleA) [reply src/shared/components/Modal/Prompt/prompt.stories.js]: done in 
e674614

**Pull request merged**